README.md

```diff
@@ -8,3 +8,61 @@ It includes Javascript and native stack traces, heap statistics,
 platform information and resource usage etc. With the report enabled,
 reports are triggered on unhandled exceptions, fatal errors, signals
 and calls to a Javascript API.
+
+Usage:
+
+    npm install nodejs/nodereport
+
+    var nodereport = require('nodereport');
+
+By default, this will trigger a NodeReport to be written to the current
```

Comment:
Hi @sam-github, you can override the defaults and configure which events you want via the API, so
`require("nodereport").setEvents("fatalerror+signal");`
will switch off the exception handler, likewise
`require("nodereport").setEvents("exception+fatalerror");`
will switch off the signal handler.

`require("nodereport").setSignal("SIGQUIT");`
will configure SIGQUIT for the trigger signal instead of SIGUSR2, and we could add more choices for that (though I just found a bug in that code, fix on it's way).